html/cross-origin-opener-policy/coep.https.html

```diff
@@ -35,10 +36,10 @@
     coop_coep_test(t, SAME_ORIGIN, variant.coop, variant.coep, variant.title.replace(/ /g,"-"), variant.opener);
   }, `Same-origin ${variant.title}`);
 
-  // This seems useful to test, CROSS_SITE is probably too redundant though.
+  // "same-site" is a historical value, equivalent to "unsafe-none"
   async_test(t => {
-    coop_coep_test(t, SAME_SITE, variant.coop, variant.coep, `same-site-${variant.title.replace(/ /g,"-")}`, false);
-  }, `Same-site ${variant.title}`);
+    coop_coep_test(t, SAME_SITE, variant.coop, variant.coep, `same-site-${variant.title.replace(/ /g,"-")}`, true);
```

Comment:
Why would this retain its opener?

Reply:
It shouldn't; the test itself has COOP: same-origin + COEP: require-corp, a SAME_SITE popup should never have an opener. Fixed.